examples/load_ui/load_ui_2.py

```diff
@@ -0,0 +1,89 @@
+import sys
+import os
+
+# Set preferred binding
+os.environ["QT_PREFERRED_BINDING"] = "PyQt4"
+
+from Qt import QtWidgets
+from Qt import __binding__
+
+
+def loadUiType(uiFile):
+    """
+    Pyside equiv for the loadUiType function in qt.
+    """
+
+    import pysideuic
+    import xml.etree.ElementTree as ElementTree
+    from cStringIO import StringIO
+
+    parsed = ElementTree.parse(uiFile)
+    widget_class = parsed.find('widget').get('class')
+    form_class = parsed.find('class').text
+
+    with open(uiFile, 'r') as f:
+        o = StringIO()
+        frame = {}
+
+        pysideuic.compileUi(f, o, indent=0)
+        pyc = compile(o.getvalue(), '<string>', 'exec')
+        exec(pyc) in frame
+
+        # Fetch the base_class and form class based on their type in
+        # the xml from designer
+        form_class = frame['Ui_%s' % form_class]
+        base_class = eval('QtWidgets.%s' % widget_class)
+    return form_class, base_class
+
+
+def loadUi(uifile, baseinstance=None, workingDirectory=None):
+    """
```

Comment:
The Google docstring syntax is:

```python
def my_function():
  """Short, one-line description

   Long description

   """

   # Code here, notice the empty line inbetween docstring and code.
```

Could we go with that, for consistency?

- http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
```